public/app/features/variables/pickers/shared/VariableLink.tsx

```diff
@@ -88,28 +90,35 @@ const LoadingIndicator: FC<Pick<Props, 'onCancel'>> = ({ onCancel }) => {
   );
 };
 
-const getStyles = (theme: GrafanaTheme) => ({
+const getStyles = (theme: GrafanaTheme2) => ({
   container: css`
     max-width: 500px;
     padding-right: 10px;
-    padding: 0 ${theme.spacing.sm};
-    background-color: ${theme.colors.formInputBg};
-    border: 1px solid ${theme.colors.formInputBorder};
-    border-radius: ${theme.border.radius.sm};
+    padding: 0 ${theme.spacing(1)};
+    background-color: ${theme.components.input.background};
+    border: 1px solid ${theme.components.input.borderColor};
+    border-radius: ${theme.shape.borderRadius(1)};
     display: flex;
     align-items: center;
     color: ${theme.colors.text};
-    height: ${theme.height.md}px;
+    height: ${theme.v1.height.md}px;
```

Comment:
Don't we have a replacement for `theme.v1` in theme v2?

Reply:
`GrafanaTheme2.v1` is like a compatibility shim that lets you opt back into the old theme for something. I initially thought that the old theme had a "better" expression of what we wanted, but you prompted to reevaluate and i updated it 😌